Kiosk/App/BidderDetailsRetrieval.swift

```diff
@@ -0,0 +1,35 @@
+import UIKit
+
+extension UIViewController {
```

Comment:
The extension is by default internal, scoped to the application. I'm adding the extension to `UIViewController` because the alert controller is presented from a view controller, and different controllers can be topmost when we want to present the alert.